src/ceph-disk/ceph_disk/main.py

```diff
@@ -799,8 +819,12 @@ def is_mounted(dev):
     """
     Check if the given device is mounted.
     """
+    if FREEBSD:
+        PROCDIR = '/compat/linux/proc'
```

Comment:
PROCDIR is already initialized at https://github.com/ceph/ceph/pull/12086/files#diff-cca3df73f74b7cc44c31596303d95b44R215. and PROCDIR is supposed to be the absolute path, so https://github.com/ceph/ceph/pull/12086/files#diff-cca3df73f74b7cc44c31596303d95b44R825 is wrong.